app/controllers/affiliations_controller.rb

```diff
@@ -0,0 +1,24 @@
+class AffiliationsController < ApplicationController
+  # POST /affiliations or /affiliations.json
+  def create
+    @affiliation = Affiliation.new(politician_id: affiliation_params[:politician_id], relationship: affiliation_params[:relationship], business_id: affiliation_params[:business_attributes][:id]) if affiliation_params[:business_attributes] && affiliation_params[:business_attributes][:id].present?
+    @affiliation = Affiliation.new(affiliation_params) if affiliation_params[:business_attributes] && affiliation_params[:business_attributes][:id].blank?
+
+    respond_to do |format|
+      if @affiliation.save
+        format.html { redirect_to business_url(@affiliation), notice: "Business was successfully created." }
```

Comment:
Is the flash message "Business was successfully created." or "Affiliation was successfully created."